client/src/shared/ui/overlay/OverlayDropdown.js

```diff
@@ -76,8 +74,8 @@ function Option(props) {
   const { text, onClick } = props;
 
   return (
-    <li>
-      <button onClick={ onClick } type="button">{ text }</button>
+    <li onClick={ onClick }>
```

Comment:
Is this part of the re-styling?

Reply:
Proposal: Always use semantic elements as click targets (`button`, `link`).

Reply:
So the idea here is that you can click anywhere on the list item and it triggers a click. Other solution for that would be to stretch the button to fill the whole space of `<li>`. Not sure if this is better. BTW this is similar to [what we already have](https://github.com/camunda/camunda-modeler/blob/develop/client/src/shared/ui/DropdownButton.js#L139).